src/platform/user/authentication/components/SignInModal.jsx

```diff
@@ -151,154 +144,102 @@ export class SignInModal extends React.Component {
+                  alt="Sign in with ID.me"
+                  src={`${vaGovFullDomain}/img/signin/idme-icon-white.svg`}
+                />
+              </button>
+              <button
+                disabled={globalDowntime}
+                type="link"
+                aria-label="Sign in with DS Logon"
+                className="usa-button usa-button-big dslogon-button vads-u-margin-y--1p5"
+                onClick={this.loginHandler('dslogon')}
+              >
+                DS Logon
+              </button>
+              <button
+                disabled={globalDowntime}
+                type="link"
+                aria-label="Sign in with My HealtheVet"
+                className="usa-button usa-button-big mhv-button vads-u-margin-y--1p5"
+                onClick={this.loginHandler('mhv')}
+              >
+                My HealtheVet
+              </button>
+              <div className="alternate-signin">
+                <h2 className="vads-u-margin-top--3">Or create an account</h2>
+                <div className="vads-u-display--flex vads-u-flex-direction--column">
+                  {this.props.loginGovEnabled && (
+                    <a
+                      role="link"
+                      tabIndex="0"
+                      aria-label="Create an account with Login.gov. Navigates to Login.gov website"
                       disabled={globalDowntime}
-                      className="idme-create usa-button usa-button-secondary"
-                      onClick={this.signupHandler}
+                      className="vads-c-action-link--blue vads-u-border-top--1px vads-u-padding-bottom--2"
+                      onClick={this.signupHandler('logingov')}
                     >
-                      <img
-                        aria-hidden="true"
-                        role="presentation"
-                        alt="ID.me"
-                        src={`${vaGovFullDomain}/img/signin/idme-icon-dark.svg`}
-                      />
-                      <strong> Create an ID.me account</strong>
-                    </button>
-                    <p>Use your email, Google, or Facebook</p>
-                  </div>
+                      Create an account with Login.gov
+                    </a>
+                  )}
+                  <a
+                    role="link"
+                    tabIndex="0"
+                    aria-label="Create an account with ID.me. Navigates to ID.me website"
+                    disabled={globalDowntime}
+                    className="vads-c-action-link--blue vads-u-border-top--1px vads-u-padding-bottom--2 vads-u-border-bottom--1px"
+                    onClick={this.signupHandler('idme')}
+                  >
+                    Create an account with ID.me
+                  </a>
                 </div>
+                {this.props.loginGovEnabled && (
+                  <a
+                    className="vads-u-display--block vads-u-margin-top--2"
+                    href="#"
```

Comment:
Same comment about using `#`

Reply:
@timwright12 Pages are going to be built and I do not have the href yet, would you like me to send this to the homepage for now?

Reply:
can we remove the link entirely until the pages are there? The link text also needs to match the destination